Source/Core/Core/HW/EXI_Device.cpp

```diff
@@ -136,7 +136,6 @@ IEXIDevice* EXIDevice_Create(TEXIDevices device_type, const int channel_num)
 		break;
 
 	case EXIDEVICE_NONE:
-	default:
```

Comment:
I'm surprised code that tests via enum arguments wouldn't sanitize against this in the first place.